src/deposit-service/AxelarDepositService.sol

```diff
@@ -17,32 +17,32 @@ contract AxelarDepositService is Upgradable, IAxelarDepositService {
     bytes32 internal constant _WRAPPED_TOKEN_SYMBOL_SLOT =
         0x91d2f5305ae2a8f5b319f6c3a690eff002c3e572220774ba5f7e957f079e55df;
 
-    bytes32 internal constant PREFIX_DEPOSIT_SEND_TOKEN = keccak256('deposit-send-token');
-    bytes32 internal constant PREFIX_DEPOSIT_SEND_NATIVE = keccak256('deposit-send-native');
-    bytes32 internal constant PREFIX_DEPOSIT_WITHDRAW_NATIVE = keccak256('deposit-withdraw-native');
+    bytes32 public constant override PREFIX_DEPOSIT_SEND_TOKEN = keccak256('deposit-send-token');
```

Comment:
Why do we want to expose these?

Reply:
For transparency, so the deposit address can be calculated offline. Do you think it's not worth it and those constants could be found in the source code?

Reply:
Removing it as we agreed